bin/deploy

```diff
@@ -2,9 +2,16 @@
 
 set -o errexit
 
+echo -n "Enter bucket_name (default: temp-serverless-resize-$(openssl rand -hex 8)):"
+read bucket_name
+
+if [ -z "$bucket_name" ]
+then
+  bucket_name="temp-serverless-resize-$(openssl rand -hex 8)"
```

Comment:
@jpignata  here if he skips the input, the default one will be set.

Reply:
Hmm. Sorry, I don't follow. This bucket is only used to stage the assets needed by SAM/CFN to install the function. It is deleted on line 35. Why would a user need to specify this temporary resource?

Reply:
Oh, I didn't notice that.

sorry, my mistake.

I will close the pull request.

Reply:
No problem at all! Thanks for opening.